<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PairDialog</class>
 <widget class="QDialog" name="PairDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>481</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>687867</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add New Pair</string>
  </property>
  <widget class="QTabWidget" name="morePairInfo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>671</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="donorCharacteristicsTab">
    <attribute name="title">
     <string>Donor Characteristics</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>211</width>
       <height>95</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="donorCharacteristicsGridLayout">
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="donorWeightSpinBox">
        <property name="maximum">
         <double>400.000000000000000</double>
        </property>
        <property name="value">
         <double>60.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="donorGenderComboBox">
        <item>
         <property name="text">
          <string>Male</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Female</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="donorHeightLabel">
        <property name="text">
         <string>Height (m)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="donorWeightLabel">
        <property name="text">
         <string>Weight (kg)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="donorGenderLabel">
        <property name="text">
         <string>Gender</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="donorHeightSpinBox">
        <property name="maximum">
         <double>3.000000000000000</double>
        </property>
        <property name="value">
         <double>1.500000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="recipCharacteristicsTab">
    <attribute name="title">
     <string>Recipient Characteristics</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>241</width>
       <height>163</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="recipCharacteristicsGridLayout">
      <item row="0" column="1">
       <spacer name="recipGenderSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="recipGenderComboBox">
        <item>
         <property name="text">
          <string>Male</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Female</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="recipWeightSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="recipHeightLabel">
        <property name="text">
         <string>Height (m)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="recipWeightLabel">
        <property name="text">
         <string>Weight (kg)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="recipHeightSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="recipRaceLabel">
        <property name="text">
         <string>Race</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="recipTODSpinBox">
        <property name="maximum">
         <double>200.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="recipRaceComboBox">
        <item>
         <property name="text">
          <string>White</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Black</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hispanic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Other</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="recipRaceSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="recipGenderLabel">
        <property name="text">
         <string>Gender</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="recipTODLabel">
        <property name="text">
         <string>Months On Dialysis</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDoubleSpinBox" name="recipHeightSpinBox">
        <property name="maximum">
         <double>3.000000000000000</double>
        </property>
        <property name="value">
         <double>1.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="recipWeightSpinBox">
        <property name="maximum">
         <double>400.000000000000000</double>
        </property>
        <property name="value">
         <double>60.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>176</width>
       <height>89</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="recipVerticalLayout">
      <item>
       <widget class="QCheckBox" name="recipHepCCheckBox">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Hepatitis C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="recipDiabetesCheckBox">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Diabetes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="recipPrevTransCheckBox">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Previous Transplant </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="hlaInfoTab">
    <attribute name="title">
     <string>HLA Information</string>
    </attribute>
    <widget class="QGroupBox" name="donorHLADRLayout">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Donor HLA DR</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>70</width>
        <height>89</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="donorHLADRVerticalLayout">
       <item>
        <widget class="QCheckBox" name="donorDR51CheckBox">
         <property name="text">
          <string>DR51</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="donorDR52CheckBox">
         <property name="text">
          <string>DR52</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="donorDR53CheckBox">
         <property name="text">
          <string>DR53</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="donorHLABWLayout">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Donor HLA BW</string>
     </property>
     <widget class="QWidget" name="layoutWidget_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>63</width>
        <height>57</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="donorHLABWVerticalLayout">
       <item>
        <widget class="QCheckBox" name="donorBW4CheckBox">
         <property name="text">
          <string>BW4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="donorBW6CheckBox">
         <property name="text">
          <string>BW6</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>268</width>
       <height>197</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="donorHLAGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="donorALabel">
        <property name="text">
         <string>Donor HLA A</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="donorA1LineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="donorA2LineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="donorBLineEdit">
        <property name="text">
         <string>Donor HLA B</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="donorB1lineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="donorB2LineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="donorCWLineEdit">
        <property name="text">
         <string>Donor HLA CW</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="donorCW1LineEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="donorCW2LineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="donorDPLabel">
        <property name="text">
         <string>Donor HLA DP</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="donorDP1LineEdit"/>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="donorDP2LineEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="donorDQLabel">
        <property name="text">
         <string>Donor HLA DQ</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="donorDQ1LineEdit"/>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="donorDQ2LineEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="donorDRLabel">
        <property name="text">
         <string>Donor HLA DR</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="donorDR1LineEdit"/>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="donorDR2LineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>641</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="recipAdditionalHorizontalLayout">
      <item>
       <widget class="QLabel" name="recipAdditionalLayout">
        <property name="text">
         <string>Recipient Antibodies</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="recipHLALineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>210</y>
       <width>361</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="donorHLAAdditionalLabel">
        <property name="text">
         <string>Additional Donor Antigens</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="donorHLALineEdit"/>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>donorHLABWLayout</zorder>
    <zorder>donorHLADRLayout</zorder>
   </widget>
   <widget class="QWidget" name="commentTab">
    <attribute name="title">
     <string>Comments</string>
    </attribute>
    <widget class="QTextEdit" name="commentTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>651</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="mainHorizontalLayout">
    <item>
     <widget class="QGroupBox" name="donorGroupBox">
      <property name="title">
       <string>Donor Information</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>251</width>
         <height>95</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="donorGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="donorNameLabel">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="donorNameLineEdit">
          <property name="text">
           <string>Donor Name</string>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="donorAgeLabel">
          <property name="text">
           <string>Age</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="donorBTLabel">
          <property name="text">
           <string>Blood Type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="donorAgeSpinBox">
          <property name="maximum">
           <number>130</number>
          </property>
          <property name="value">
           <number>40</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="donorAgeSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <spacer name="donorBTSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="donorBTComboBox">
          <item>
           <property name="text">
            <string>O</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AB</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="recipGroupBox">
      <property name="title">
       <string>Recipient Information</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>251</width>
         <height>129</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="recipGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="recipNameLabel">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="recipNameLineEdit">
          <property name="text">
           <string>Recipient Name</string>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="recipAgeLabel">
          <property name="text">
           <string>Age</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="recipAgeSpinBox">
          <property name="maximum">
           <number>130</number>
          </property>
          <property name="value">
           <number>40</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="recipAgeSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="recipBTLabel">
          <property name="text">
           <string>Blood Type</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="recipBTComboBox">
          <item>
           <property name="text">
            <string>O</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AB</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2">
         <spacer name="recipBTSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="recipPRALabel">
          <property name="text">
           <string>PRA</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="recipPRASpinBox">
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSlider" name="praSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>112</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>morePairInfo</zorder>
  <zorder>buttonBox</zorder>
 </widget>
 <tabstops>
  <tabstop>donorNameLineEdit</tabstop>
  <tabstop>donorAgeSpinBox</tabstop>
  <tabstop>donorBTComboBox</tabstop>
  <tabstop>recipNameLineEdit</tabstop>
  <tabstop>recipAgeSpinBox</tabstop>
  <tabstop>recipBTComboBox</tabstop>
  <tabstop>recipPRASpinBox</tabstop>
  <tabstop>praSlider</tabstop>
  <tabstop>morePairInfo</tabstop>
  <tabstop>donorGenderComboBox</tabstop>
  <tabstop>donorWeightSpinBox</tabstop>
  <tabstop>donorHeightSpinBox</tabstop>
  <tabstop>recipGenderComboBox</tabstop>
  <tabstop>recipRaceComboBox</tabstop>
  <tabstop>recipTODSpinBox</tabstop>
  <tabstop>recipWeightSpinBox</tabstop>
  <tabstop>recipHeightSpinBox</tabstop>
  <tabstop>recipHepCCheckBox</tabstop>
  <tabstop>recipDiabetesCheckBox</tabstop>
  <tabstop>recipPrevTransCheckBox</tabstop>
  <tabstop>recipHLALineEdit</tabstop>
  <tabstop>donorA1LineEdit</tabstop>
  <tabstop>donorA2LineEdit</tabstop>
  <tabstop>donorB1lineEdit</tabstop>
  <tabstop>donorB2LineEdit</tabstop>
  <tabstop>donorCW1LineEdit</tabstop>
  <tabstop>donorCW2LineEdit</tabstop>
  <tabstop>donorDP1LineEdit</tabstop>
  <tabstop>donorDP2LineEdit</tabstop>
  <tabstop>donorDQ1LineEdit</tabstop>
  <tabstop>donorDQ2LineEdit</tabstop>
  <tabstop>donorDR1LineEdit</tabstop>
  <tabstop>donorDR2LineEdit</tabstop>
  <tabstop>donorDR51CheckBox</tabstop>
  <tabstop>donorDR52CheckBox</tabstop>
  <tabstop>donorDR53CheckBox</tabstop>
  <tabstop>donorBW4CheckBox</tabstop>
  <tabstop>donorBW6CheckBox</tabstop>
  <tabstop>donorHLALineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>recipPRASpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>praSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>praSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>recipPRASpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PairDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PairDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
